scripts/llogcolor

```diff
@@ -152,9 +190,26 @@ def main():
 
             if options.date:
                 ts = str(datetime.datetime.fromtimestamp(float(ts)))
-                line = thread_color[tid] + ts + ' ' + result.group('begin') + result.group('middle') + tid + result.group('end') + '\n'
+                line = (
+                    thread_color[tid]
+                    + ts
```

Comment:
The leading `+` here is in the style guide. The rational is that it's more consistent with math notation, although in this case it's concatenation, not arithmetic.